.github/workflows/command-l10n-update.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         if: ${{ needs.init.outputs.arg1 != 'fixup' && needs.init.outputs.arg1 != 'amend' }}
         run: |
           git add .
-          git commit --signoff -m 'Updating l10n asset'
+          git commit --signoff -m 'chore(l10n): extracted new translations'
```

Comment:
👀
```suggestion
          git commit --signoff -m 'chore(l10n): extract new translations'
```

Reply:
both seem ok to me?

Reply:
To follow convention, to say "what it does when it is committed (applied)" and not what was done.

Reply:
makes sense :)